apps/system/utils/kdbg_date.c

```diff
@@ -247,6 +302,21 @@ int kdbg_date(int argc, char **args)
 		printf("                'month', 'day', 'hour':'minute':'second' 'year'\n");
 		printf("                Example: Apr 21 10:35:22 1991\n");
 		ret = ERROR;
+#else
+		printf("\nUsage: date\n");
+		printf("   or: date [-s FORMAT]\n");
+		printf("Display, or Set system time and date information\n");
+		printf("\nOptions:\n");
+		printf(" -s FORMAT     Set system time in the given FORMAT\n");
+		printf("               FORMAT: MMM DD HH:MM:SS YYYY DWK DYR D\n");
```

Comment:
1. I guess DYR is "Day of Year Range". But why "Day of Week Range" is DWK?
2. In date_settime, you compare string. But here it looks we can use a number, not string.